mod/filter/set/abstract/browse_filter_form.rb

```diff
@@ -45,19 +31,17 @@ def target_type_id
   WikirateCompanyID
 end
 
-def get_query params={}
-  search_args = filter_wql
-  add_sort_wql search_args, sort_param if sort?
-  params[:query] = search_args
-  super(params)
+def wql_hash
+  wql = filter_wql
+  wql[:limit] = 20
+  add_sort_wql wql, sort_param if sort?
+  wql
```

Comment:
part of the reason I had "raw" in the name of this method was that I was thinking this method was generally *not* supposed to have limits and other things that are frequently varied in it.  maybe "core_query" would be better?

In any event, does it break our pattern to put the limit in here?